README.md

```diff
@@ -724,885 +711,586 @@ signIn.after('identify', async () => {
-     // 2) onSuccess: success callback.
-     // 3) onFailure: failure callback. Note: accepts an errorObject that can be used to show form level
-     //    or field level errors.
-    postSubmit: function (response, onSuccess, onFailure) {
-      // In this example postSubmit callback is used to log the server response to the browser console before completing registration flow
-      console.log(response);
-      // call onSuccess to finish registration flow
-      onSuccess(response);
-    }
-    ```
-- **onFailure and ErrorObject:** The onFailure callback accepts an error object that can be used to show a form level vs field level error on the registration form.
+##### helpLinks.forgotPassword
 
-    ####  Use the default error
-    ```javascript
-    preSubmit: function (postData, onSuccess, onFailure) {
-      // A Default form level error is shown if no error object is provided
-      onFailure();
-    }
-    ```
+Custom link href for the "Forgot Password" link
 
-    #### Use form level error
-     ```javascript
-    preSubmit: function (postData, onSuccess, onFailure) {
-      var error = {
-        "errorSummary": "Custom form level error"
-      };
-      onFailure(error);
-    }
-    ```
+##### helpLinks.unlock
 
-    #### Use field level error
-    ```javascript
-      preSubmit: function (postData, onSuccess, onFailure) {
-        var error = {
-            "errorSummary": "API Error",
-            "errorCauses": [
-                {
-                    "errorSummary": "Custom field level error",
-                    "reason": "registration.error.address",
-                    "resource": "User",
-                    "property": "address", //should match field name
-                    "arguments": []
-                }
-            ]
-        };
-        onFailure(error);
-     }
-    ```
+Custom link href for the "Unlock Account" link. For this link to display, `features.selfServiceUnlock` must be set to `true`, and the self service unlock feature must be enabled in your admin settings.
 
-### IdP Discovery
+##### helpLinks.custom
 
-IdP Discovery enables you to route users to different 3rd Party IdPs that are connected to your Okta Org. Users can federate back into the primary org after authenticating at the IdP.
+Array of custom link objects `{text, href, target}` that will be added to the *"Need help signing in?"* section. The `target` of the link is optional.
 
-To use IdP Discovery in your application, configure an [identity provider routing rule](https://help.okta.com/en/prod/Content/Topics/Security/configure-routing-rules.htm) in the Okta admin panel.
-Then, in the widget configuration, set `features.idpDiscovery` to `true`.
+- **features.autoFocus** - Automatically focuses the first input field of any form when displayed. Defaults to `true`.
 
-```javascript
-var signIn = new OktaSignIn({
-  baseUrl: 'https://{yourOktaDomain}',
-  ... ...
-  features: {
-    idpDiscovery: true
-  }
-});
-```
+### Hooks
```

Comment:
maybe as follow up - list all support hook name for both `before` and `after` ?

Reply:
This would be the last of all possible remediations/form names. Would be nice to have this documented I agree!